lib/reek/cli/command/todo_list_command.rb

```diff
@@ -12,17 +12,19 @@ module Command
       # file that can serve as a todo list.
       #
       class TodoListCommand < BaseCommand
-        FILE_NAME = '.todo.reek'
+        DEFAULT_CONFIGURATION_FILE_NAME = Configuration::ConfigurationFileFinder::DEFAULT_FILE_NAME
+        DETECTOR_KEY = Configuration::AppConfiguration::DETECTORS_KEY
```

Comment:
This level of indirection seems undesirable to me. I have a feeling that if it seems necessary, it probably makes sense for these constants to live in some module where they're readily accessible for all modules.